bundle/metadata/dependencies.yaml

```diff
@@ -2,4 +2,4 @@ dependencies:
   - type: olm.package
     value:
       packageName: openshift-pipelines-operator-rh
-      version: "1.7.1"
+      version: ">=1.5.2"
```

Comment:
We're supporting older versions of openshift-pipelines-operator-rh with this directive, right? Is that the goal?

Reply:
This is a great question. Since we have a hard dependency on the pipelines operator, we need to may sure any version greater then 1.5.2 is in the cluster (since this is the version that has the CRD's we rely on). Ideally, as pipelines operator released new versions we'd update to the latest version, but they do not release all new versions to all OpenShift versions. So to make sure that OCO (this operator) works on all clusters we need to support 4.8 onward, this syntax makes sure we get the newest version in whatever cluster runs this operator. Hope this is clear.

Reply:
Got it, and the common alternative to this approach is to release multiple streams, each with a hard-pinned dependency vs. this which is more flexible but releases a single stream of this project?

Reply:
Correct. I'm not sure how we'd go to a multi-stream approach with a hard pinned version at this point(in a logistical sense), nor do I think we want that kind of overhead when trying to certify.